SportHub/src/modules/UserHeader/style/style.module.scss

```diff
@@ -0,0 +1,178 @@
+@import "src/styles/mixins";
+
+select::-ms-expand {
+  display: none;
+}
+
+  .header{
+      width: 100vw;
+      height: 12vh;
+      border-bottom: 1px solid rgb(240, 240, 240);
+      .logo{
+        position: absolute;
+        left:0;
+        top:0;
+        @include flex-center;
+        background: var(--color-red-regular);
+        width: 18vw;
+        height: 12vh;
+        font-size: 22px;
+        color: white;
+
+        .logo_text{
+          &:hover{
+            cursor: pointer;
+          }
+        }
+      }
+      .searchbar{
+          position: absolute;
```

Comment:
again, position absolute only where it's really needed because it cause unexpected behaviour